docs/sdk-and-tools/chain-simulator.md

```diff
@@ -0,0 +1,34 @@
+---
+id: chain-simulator
+title: Chain simulator
+---
+
+[comment]: # (mx-context-auto)
+
+## Overview
+
+Chain simulator is a binary that provides all the `mx-chain-proxy-go` endpoints and includes additional endpoints 
```

Comment:
"is a piece of software" - instead of "a binary".

Reply:
Would be nice to add extra notes on how is the chain simulator useful for web3 developers. And perhaps even the fact that it's integrated in xSuite?

Reply:
added more information